tests/e2e/iks-cluster

```diff
@@ -378,6 +378,11 @@ function cluster_create_vpc {
     if [[ "$K8S_PLATFORM" == "openshift" ]]; then
         # OpenShift minimum 2 worker nodes are required
         worker_node_count=$WORKER_NODE_COUNT
+        if [[ "$e2e_env" == "prod" ]]; then
+            E2ETEST_IC_COS_INSTANCE=$(ibmcloud resource service-instance e2e-test-roks  | grep "^ID:" | awk '{print $2}')
```

Comment:
Do we need to delete this cos instance or retain ? something we do with cluster and Retain option ?

Reply:
We are reusing COS instance.